src/test/java/org/apache/sysds/test/TestUtils.java

```diff
@@ -62,9 +62,9 @@
  * <li>clean up</li>
  * </ul>
  */
-public class TestUtils 
+public class TestUtils
```

Comment:
I think you might have accidentally reformatted TestUtils.
This needs to be done, but i would rather do it in a single PR/Commit.